### PR TITLE
Feature/update info

### PR DESCRIPTION
**Pull request by Frank-Wiebeler**
Add Info for software udates

### CODE REVIEW

File: docs/integration/authorization.adoc [hunk 1]
```diff
@@ -224,7 +224,7 @@ The result token is a base64 encoded JSON object including the following paramet
 |=======================================================================================================================================================
 |Name |Type |Description
 |account |String |The unique id of the user account on agrirouter , that will be provided to you in the metrics exports for billing
-|regcode |String |If response-type=onboard, this regcode will deliver a Registration Code equal to clicking the generate TAN-Button in the agrirouter ui
+|regcode |String |If response-type=onboard, this regcode will deliver a Registration Code equal to clicking the generate RegistrationCode-Button in the agrirouter ui
```
Comment (saschadoemer) [inline docs/integration/authorization.adoc]: From my point of view - there"s only a "Telemetry-Connection" button, which adds a telemetry connection and then generates a registration code. Could be more precise.

File: docs/integration/authorization.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ As the authorization process – in difference to the rest of the communication
 
 The authorization process is major for farming software and telemetry platforms, for CUs it is an optional function.
 
-For CUs, this requires additional infrastructure, therefore DKE advices to use the TAN exchange process.
+For CUs, this requires additional infrastructure, therefore DKE advices to use the Registration Code exchange process.
```
Comment (saschadoemer) [inline docs/integration/authorization.adoc]: From my point of view - "Registration Code" should be written as "registration code"

File: docs/update.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+:toc:
+:toc-title:
+:toclevels: 4
+
+== General
+Changing the software version of your app might require a change in agrirouter, if you decide to add new technical message types.
+
+=== When is a new certification version required?
+agrirouter is not interested in any internal changes of your software.
+However, whenever the maximum capabilities of your software change, you'll need to create a new version in the agrirouter UI and run a new certification.
```
Comment (saschadoemer) [inline docs/update.adoc]: Same when there a capabilty which is no longer supported - my point of view.

File: docs/update.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+=== What has to be done?
+==== While development
+1. You have to create a new software version in the agrirouter UI
```
Comment (saschadoemer) [inline docs/update.adoc]: New "certification version" to use common terms.

File: docs/update.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+2. After testing, a certification has to be done
+
+==== On deployment and after Software Update
```
Comment (saschadoemer) [inline docs/update.adoc]: "software upate"

File: docs/update.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+After update, the app instance has to inform agrirouter about its new capabilities by sending its new capabilities including the new certificationVersionId.
```
Comment (saschadoemer) [inline docs/update.adoc]: "including the ID of the new certification version." - we should not mix unformatted technical terms with common language.

File: docs/update.adoc [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+=== Safe updates
+When updating your software, the OnboardingResponse received from link:./integration/onBoarding.adoc[onboarding] must be stored so that it is not deleted during update.
```
Comment (saschadoemer) [inline docs/update.adoc]: Same as above.

Comment (Frank-Wiebeler) [reply docs/update.adoc]: changed

File: docs/update.adoc (hunk 6 above)
Comment (Frank-Wiebeler) [reply docs/update.adoc]: adjusted

File: docs/update.adoc (hunk 5 above)
Comment (Frank-Wiebeler) [reply docs/update.adoc]: adjusted

File: docs/update.adoc (hunk 4 above)
Comment (Frank-Wiebeler) [reply docs/update.adoc]: Agree, done

File: docs/update.adoc (hunk 3 above)
Comment (Frank-Wiebeler) [reply docs/update.adoc]: Checking with DKE

File: docs/integration/authorization.adoc (hunk 2 above)
Comment (Frank-Wiebeler) [reply docs/integration/authorization.adoc]: adjusted

File: docs/integration/authorization.adoc (hunk 1 above)
Comment (Frank-Wiebeler) [reply docs/integration/authorization.adoc]: Wording adjusted

File: docs/update.adoc (hunk 3 above)
Comment (Frank-Wiebeler) [reply docs/update.adoc]: Clarified by DKE; should not be changed

**Pull request merged**